addons/default/visiosoft/json-field_type/src/JsonFieldType.php

```diff
@@ -5,10 +5,5 @@
 class JsonFieldType extends FieldType
 {
 
-    protected $inputView = 'visiosoft.field_type.json::input';
```

Comment:
why do we need to delete it? @diashalabi @spektra2147

Reply:
this option is to overwrite the default view for the input used in the admin panel, we don't have a custom view so there is an error when you try to display the input, ([input] view not found), so we don't actually need it since we are not overwritting the default view